app/coffee/Features/Announcements/AnnouncementsHandler.coffee

```diff
@@ -5,20 +5,44 @@ _ = require("lodash")
 logger = require("logger-sharelatex")
 settings = require("settings-sharelatex")
 
-module.exports =
 
-	getUnreadAnnouncements : (user_id, callback = (err, announcements)->)->
+
+
+module.exports = AnnouncementsHandler = 
+
+
+
+	_domainSpecificAnnouncements : (email)->
+		domainSpecific = _.filter settings?.domainAnnouncmentsToShow, (domainAnnouncment)->
```

Comment:
domainAnnouncmentsToShow -> domainAnnounc**e**mentsToShow. Wouldn't normally care, but this is a settings file entry